plugins/generator-1.16.4/forge-1.16.4/procedures/get_dimensionid.java.ftl

```diff
@@ -0,0 +1,9 @@
+<#if field$dimension=="Surface">

```

Comment:
This was not tested at all as dimensions no longer have IDs.

Reply:
Please check dimension PR which implements this properly. (you can probably remove this file thus as that PR adds it)

Reply:
Done. I deleted the file.